test/integration/targets/win_iis_webapplication/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Cannot use win_feature to install IIS on Server 2008.
+# Run a brief check and skip hosts that don't support
+# that operation
+- name: check if win_feature will work on test host
```

Comment:
Why aren't you simply checking the Powershell / Os version using facts?

Reply:
I had stolen this from the win_iis_webapppool tests. Since we're running into issues I'll definitely rework it to use facts.

Reply:
using ansible facts now